packages/wallets/webpack.config.js

```diff
@@ -41,14 +41,20 @@ module.exports = function (env) {
         },
         externals: [
             {
+                '@deriv/quill-icons': `@deriv/quill-icons`,
+                '@deriv/shared': '@deriv/shared',
+                '@deriv/utils': '@deriv/utils',
                 '@deriv-com/analytics': '@deriv-com/analytics',
                 '@deriv-com/translations': '@deriv-com/translations',
+                '@deriv-com/utils': '@deriv-com/utils',
                 classnames: true,
-                moment: true,
+                moment: 'moment',
```

Comment:
Is this still required if we've removed any dependency on moment?

Reply:
@nijil-deriv true this can be removed and it does not affect the bundle size. Changed this before doing the other optimizations and forgot to remove it afterwards. Updated 👍